ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/js/button.js

```diff
@@ -0,0 +1,33 @@
+$(function() {
+            let userData = getUserCookie("hcluser");     
+            if(userData != "") {
+                //hide signin, create account buttons
+                //enable logout button
+                $('.btn-cmp-signin').css('display','none');
+                $('.btn-cmp-signup').css('display','none');
+                $('.btn-cmp-logout').css('display','inline-block');
+                let html = '<span class="btn-cmp-logout-text">Hi, '+ JSON.parse(userData).firstname +'</span>';
```

Comment:
@rakeshkumar-verma @umeshv12 : How are we handling the scenrio in case userData is not present